liquidctl/driver/nzxt_smart_device.py

```diff
@@ -163,29 +217,112 @@ def get_status(self, **kwargs):
+            for x in range(12):     # we may not get reply right away, so try up to 12 times
+                if numValidRepliesReceived == 4:
+                    break   
+                msg = self.device.read(_READ_LENGTH_V2)
+                LOGGER.debug('received %s', ' '.join(format(i, '02x') for i in msg))
+                if msg1101Reply == False and msg[0] == 0x11 and msg[1] == 0x01:  # the correct reply must start with 0x11, 0x01
+                    fw = '{}.{}.{}'.format(msg[0x11], msg[0x12], msg[0x13])
+                    status.append(('Firmware:', fw, ''))
+                    numValidRepliesReceived += 1 
+                    msg1101Reply = True
+                    continue
+                if msg2103Reply == False and msg[0] == 0x21 and msg[1] == 0x03:  # the correct reply must start with 0x21, 0x03
+                    ledStripCount = 0
+                    ledUnderglowCount = 0
+                    ledCableCombCount = 0
+                    ledAerFan120Count = 0
+                    ledAerFan140Count = 0
+                    for ledIter in range(15):
+                        ledCode = msg[0x0f + ledIter]
+                        if ledCode == 0x04:
+                            ledStripCount += 1
+                            ledName = '  LED Strip #{}'.format(ledStripCount)
+                        elif ledCode == 0x08:
+                            ledCableCombCount += 1
+                            ledName = ' Cable Comb #{}'.format(ledCableCombCount)
+                        elif ledCode == 0x0a:
+                            ledUnderglowCount += 1
+                            ledName = '  Underglow #{}'.format(ledUnderglowCount)
+                        elif ledCode == 0x0b:
+                            ledAerFan120Count += 1
+                            ledName = 'AER Fan 120 #{}'.format(ledAerFan120Count)
+                        elif ledCode == 0x0c:
+                            ledAerFan140Count += 1
+                            ledName = 'AER Fan 140 #{}'.format(ledAerFan140Count)
+                        if ledCode != 0:
+                            status.append(('    HUE2:', ledName, ''))
+                    numValidRepliesReceived += 1
+                    msg2103Reply = True
+                    continue
+                if msg6702Reply == False and msg[0] == 0x67 and msg[1] == 0x02:
+                    rpm_offset = 24
+                    duty_offset = 40
+                    noise_offset = 56
+                    for i, _ in enumerate(self._speed_channels):
+                        # LOGGER.debug('Iteration {}'.format(i))
+                        if ((msg[rpm_offset] != 0x0) and (msg[rpm_offset+1] != 0x0)): 
+                            status.append(('   Speed: Fan {}'.format(i+1), msg[rpm_offset+1] << 8 | msg[rpm_offset], 'rpm'))
+                            status.append(('    Duty: Fan {}'.format(i+1), msg[duty_offset+i], '%'))
```

Comment:
While there might be issues with this choice, the liquidctl drivers (including this one) already adopt the convention of avoiding pseudo-keys (e.g. no `Speed: Fan`).  The following would be more in line with that convention:

```python
('Fan {} speed', some_value, 'rpm')
('Fan {} duty', some_value, '%')
```

Also, remove the leading spaces.

Reply:
Done. Here's a sample output:
```
Device 1, NZXT Smart Device V2
Fan 2 duty                    40  %  
Fan 2 speed                  875  rpm
Firmware:                  1.5.0     
LED 1 accessory 1     HUE 2 LED Strip     
LED 1 accessory 2     HUE 2 Underglow     
LED 1 accessory 3     HUE 2 Underglow     
LED 2 accessory 1     AER RGB 2 140 mm     
LED 2 accessory 2     AER RGB 2 140 mm     
LED 2 accessory 3     AER RGB 2 140 mm     
LED 2 accessory 4     AER RGB 2 120 mm     
Noise level                   66  dB 
```